snapshot/bundlestore/http_store.go

```diff
@@ -10,19 +10,25 @@ import (
 	"time"
 
 	log "github.com/Sirupsen/logrus"
+	"github.com/sethgrid/pester"
 )
 
 func MakeHTTPStore(rootURI string) Store {
 	if !strings.HasSuffix(rootURI, "/") {
 		rootURI = rootURI + "/"
 	}
-	client := &http.Client{Timeout: 30 * time.Second}
+	client := pester.New()
+	client.Backoff = pester.ExponentialBackoff
+	client.MaxRetries = 8
```

Comment:
I agree with Ryan.  Putting these sorts of constants (ones that impact performance) in a common section, (typically at the top of a file),  makes it easy to identify where we may want to tune the system.  We don't have to externalize them immediately.